main.tf

```diff
@@ -429,11 +483,15 @@ resource "google_cloud_run_v2_job" "agentless_orchestrate" {
         }
         env {
           name  = "GCP_SCAN_LIST"
-          value = join(", ", var.project_filter_list)
+          value = join(", ", local.project_filter_list)
         }
         env {
           name  = "GCP_CUSTOM_SUBNETWORK"
-          value = var.custom_vpc_subnet
+          value = local.region_subnetwork_id
+        }
+        env {
+          name  = "USE_PUBLIC_IPS"
```

Comment:
@ammarekbote Let me know if you think this env var is acceptable.  I think we can probably make the use of public IPs optional for the AWS version of the module as well, so I wanted to it be agnostic rather than starting with `GCP_` but let me know what you think.